pkg/arvo/app/aqua.hoon

```diff
@@ -507,28 +505,42 @@
   ?-  -.ae
   ::
       %init-ship
-    ::  XX  Note that the keys that get passed in are unused. The keys field
-    ::      should be deleted now that aqua is capable of managing azimuth state
-    ::      internally. Its been left this way for now until all the ph tests
-    ::      can be rewritten
-    =/  keys=dawn-event:jael  (dawn who.ae)
     =.  this  abet-pe:(publish-effect:(pe who.ae) [/ %sleep ~])
     =/  initted
       =<  plow
       %-  push-events:apex:(pe who.ae)
       ^-  (list unix-event)
-      :~  [/ %wack 0]  ::  eny
-          [/ %whom who.ae]  ::  eny
-          [//newt/0v1n.2m9vh %born ~]
-          [//behn/0v1n.2m9vh %born ~]
-          :^  //term/1  %boot  &
-          ?~  keys.ae
-            [%fake who.ae]
-          [%dawn keys]
-          -.userspace-ova.pil
-          [//http-client/0v1n.2m9vh %born ~]
-          [//http-server/0v1n.2m9vh %born ~]
-          [//http-server/0v1n.2m9vh %live 8.080 `8.445]
+      %-  zing
+      :~
+        :~  [/ %wack 0]  ::  eny
+            :: [/ %verb `|]  :: possible verb
+            :^  /  %wyrd  [%nonce /a-version] :: dummy runtime version + nonce
```

Comment:
This should probably at least be `[~.nonce /aqua]`. To match vere's behavior, the nonce would be `(scot %ta (mug now.bowl))`.